WordPress/gutenberg
# Site Editor: Fix insufficient height of white background in the code editor

## What?
This PR fixes a problem in the code editor of the site editor where the white background is cut off in the middle when you have a lot of content in the code editor.

![before](https://github.com/WordPress/gutenberg/assets/54422211/077bf2e4-380c-4259-820e-c0a8dd1c3e95)

## Why?
The element with this background color is `div.edit-site-code-editor`, which is a child of Flexbox, so `min-height:100%` does not extend to the full height of the parent element.

## How?
Changed to `flex-grow:1` as well as [the post editor](https://github.com/WordPress/gutenberg/blob/115e7b18157c8b6434aa4272d540955557336031/packages/edit-post/src/components/text-editor/style.scss#L2-L5).

## Testing Instructions

Confirm that the white background color extends to full height when you have a lot of content.

![after_1](https://github.com/WordPress/gutenberg/assets/54422211/d814fdda-0b30-4e78-8c5e-1d7714b480b3)

Similarly, even if there is little content, confirm that the white background color extends the full height.

![after_2](https://github.com/WordPress/gutenberg/assets/54422211/7d3e94a7-c8ed-4ac2-83d9-5b8541b25545)
